spec/features/merchant/show_spec.rb

```diff
@@ -62,4 +64,37 @@
       expect(page).to have_content(@item_1.name)
     end
   end
+
+  describe "as an Admin" do
+    before :each do
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@admin)
+    end
+
+    it "can see merchant dashboard as an admin and see everything a merchant can see" do
+      visit "/merchants"
+      expect(page).to have_content(@bike_shop.name)
+      expect(page).to have_link(@bike_shop.name)
+      click_link "#{@bike_shop.name}"
+      expect(current_path).to eq("/admin/merchants/#{@bike_shop.id}")
+
+      expect(page).to have_content(@bike_shop.name)
+      expect(page).to have_link(@bike_shop.name)
+      expect(page).to have_content(@bike_shop.address)
+      expect(page).to have_content(@bike_shop.city)
+      expect(page).to have_content(@bike_shop.state)
+      expect(page).to have_content(@bike_shop.zip)
+      expect(page).to have_content("123 Bike Rd.\nDenver, CO 80203")
+
+      expect(page).to have_content(@order_1.id)
+      expect(page).to have_link(@order_1.id)
+      expect(page).to have_content(@order_1.created_at)
+      expect(page).to have_content(@item_order_1.quantity)
+      expect(page).to have_content(@item_order_1.quantity * @item_order_1.price)
+
+      expect(page).to have_link("Items for #{@bike_shop.name}")
+      click_link "Items for #{@bike_shop.name}"
+      expect(current_path).to eq("/admin/merchants/#{@bike_shop.id}/items")
+      expect(page).to have_content(@item_1.name)
+    end
+  end
```

Comment:
Great job breaking this up into some more describe blocks.

Reply:
Woo! Thank you for teaching me that we CAN use these nested describes and before :eaches